samples/msal-node-samples/custom-INetworkModule-and-network-tracing/package.json

```
@@ -4,14 +4,17 @@
   "description": "Custom INetworkModule Sample & Network Tracing via Fiddler Everywhere",
   "scripts": {
     "build": "npx tsc",
-    "start": "npm run build && node dist/app.js"
+    "start": "npm run build && node dist/app.js",
+    "build:package": "cd ../../../lib/msal-common && npm run build && cd ../msal-node && npm run build",
+    "install:common": "npm install ../../../lib/msal-common",
+    "install:node": "npm install ../../../lib/msal-node"
```

```suggestion
    "install:common": "npm install ../../../lib/msal-common",
    "install:node": "npm install ../../../lib/msal-node"
```

nit: might be handier to join these two scripts, i.e. install: local